lib/rules.js

```diff
@@ -222,7 +222,16 @@ var rules = {
   'isURL': {
     fn: function(x, opt) {
       if (typeof x !== 'string') { throw new Error ('Value was not a string.'); }
-      return validator.isURL(x, opt === true ? undefined : opt);
+      var defaultOptions = {};
+      // As of Validator 8.0.0, isURL requires `require_tld` to be set to false to validate localhost as a URL.
+      if(x.match(/^(https?:\/\/)?localhost(\:|$)/)) {
+      // If the value we're checking is localhost, we'll add require_tld: false to the options we pass into validator
+        defaultOptions = {
+          require_tld: false,//eslint-disable-line camelcase
+        };
+      }
+      var options = _.merge(defaultOptions, opt); // Note: If the provided options include `require_tld: true`, that value will override the value set in defaultOptions.
```

Comment:
Any reason this is a deep merge instead of an _.extend()?

(Deep merges are gnarly and worth avoiding most of the time.)